### PR TITLE
GeoNet NZ Quakes feed integration

### PR DESCRIPTION
**Pull request by exxamalte**
**Description:**
This documents the new GeoNet NZ Quakes feed integration.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#25736

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

<a href="https://gitpod.io/#https://github.com/home-assistant/home-assistant.io/pull/10077"></a>

### CODE REVIEW

File: source/_components/geo_location.markdown [hunk 1]
```diff
@@ -16,6 +16,7 @@ The [Geolocation trigger](/docs/automation/trigger/#geolocation-trigger) can be
 | Platform                                          | Source                        |
 |---------------------------------------------------|-------------------------------|
 | GeoJSON Events                                    | `geo_json_events`               |
+| GeoGeoNet New Zealand Quakes                      | `geonetnz_quakes`               |
```
Comment (MartinHjelmare) [inline source/_components/geo_location.markdown]: It should be GeoNet, right?

**Review by MartinHjelmare — APPROVED** [review]

**Review by MartinHjelmare — CHANGES_REQUESTED** [review]
Just the comment above.

File: source/_components/geo_location.markdown (hunk 1 above)
Comment (exxamalte) [reply source/_components/geo_location.markdown]: Oops. Fixed typo.

**Review by MartinHjelmare — APPROVED** [review]

**Review by frenck — APPROVED** [review]
Looks great @exxamalte! Thanks! 👍

**Pull request merged**